ee/vellum_ee/workflows/display/nodes/vellum/api_node.py

```diff
@@ -211,6 +211,11 @@ def serialize(
                 "status_code_output_id": str(status_code_output_display.id),
             },
             **self.serialize_generic_fields(display_context),
+            "outputs": [
+                {"id": str(text_output_display.id), "name": "text", "type": "STRING", "value": None},
+                {"id": str(json_output_display.id), "name": "json", "type": "JSON", "value": None},
+                {"id": str(status_code_output_display.id), "name": "status_code", "type": "NUMBER", "value": None},
+            ],
```

Comment:
this should come from `.serialize_generic_fields`. for all nodes not ready (so all nodes that call `serialize_generic_fields` but is not the api node), pass in a `exclude` kwarg which is a list of keys to pop